src/sentry/static/sentry/app/views/organizationReleases/list/layout.jsx

```diff
@@ -0,0 +1,36 @@
+import styled from 'react-emotion';
+
+import space from 'app/styles/space';
+
+const Layout = styled('div')`
+  display: grid;
+  grid-template-columns: 4fr 1fr 1fr 1fr;
+  grid-column-gap: ${space(1.5)};
+  width: 100%;
+  align-items: center;
+  grid-template-areas: 'release-name stats new-count last-event';
+
+  @media (max-width: ${p => p.theme.breakpoints[0]}) {
+    grid-template-columns: 4fr 1fr 1fr;
+    grid-template-areas: 'release-name new-count last-event';
+  }
```

Comment:
I prefer this over the grid-emotion syntax. This is much more direct and simple to understand over the `/` syntax. :clap:

Reply:
I just looked at the `grid-emotion` package; it is now deprecated 😬  https://github.com/emotion-js/grid-emotion

Reply:
pretty much anything is easier to understand than grid emotion syntax lol

Reply:
> it is now deprecated

It's been deprecated for at least a year now, we just haven't been able to move over to https://github.com/rebassjs/grid